Gopkg.toml

```diff
@@ -78,6 +78,11 @@ required = [
   name = "k8s.io/client-go"
   version = "kubernetes-1.12.5"
 
+[[override]]
+  name = "k8s.io/gengo"
+  # HEAD as of 2019-03-27
+  revision = "e17681d19d3ac4837a019ece36c2a0ec31ffe985"
```

Comment:
I'm slightly concerned about pinning here as various Kubernetes components move forwards.

On the other hand, it looks like gengo has no formal release process, so there's nothing that we can particularly pin to, and we previously vendored this [on 29 May, 2018](https://github.com/kubernetes/gengo/commit/2e1a79edcaecf0bfbde129a1fd55624b66adb699).

Try running `dep ensure -update k8s.io/gengo` without modifying Gopkg.toml to see if it will update the dependency without needing to pin it.

You might also want to run `./hack/update-deps.sh` and `./hack/update-codegen.sh` along with the dependency update, or tests may fail.

Reply:
Dropped the `Gopkg.toml` change.